dao/src/main/java/cz/idomatojde/entity/User.java

```diff
@@ -39,7 +38,7 @@
     @NotNull
     private String passSalt;
 
-    @Column(nullable=false,unique=true)
+    @Column(nullable = false,unique = true)
     @Pattern(regexp=".+@.+\\....?")
```

Comment:
Ehm, I meant the space after the comma, this way it is worse since the very next line is still without spaces lel. Either add it everywhere or just nowhere. IDEA's quick format should handle this the right way.